app/src/main/java/com/hamdan/forzenbook/di/login/LoginModule.kt

```diff
@@ -64,6 +66,11 @@ object LoginModule {
         //return LoginGetTokenUseCaseImpl(loginRepository)
     }
 
-    private const val LOGIN_BASE_URL = "https://forzen.dev/"
+    @Provides
+    fun providesLoginRequestValidationUseCase(loginRepository: LoginRepository): LoginRequestValidationUseCase {
+        return LoginRequestValidationUseCaseImpl(loginRepository)
+    }
+
+    private const val LOGIN_BASE_URL = "https://forzen.dev/api/"
     private const val REPO_NAME = "login"
```

Comment:
?? Not needed

Reply:
the duplicated item or the provideing of the impl

Reply:
What is the REPO_NAMe supposed to be for here?

Reply:
it's meant to help distinguish between the retrofit objects for dagger otherwise it will be like yo dude wtf theres two retrofits which u want?
The name is not ideal though, renamed it to MODULE_NAME